src/models/metadata.ts

```diff
@@ -260,7 +260,7 @@ export class Metadata {
   }
 
   @Memoize() get noindex(): BooleanField | undefined {
-    return this.rawMetadata?.noindex
+    return this.rawMetadata?.noindex != null
```

Comment:
Hmm, I think my rationale for this change was that `false` values were being incorrectly parsed as undefined (which the new suggestion also addresses). But if we want all the fields to pass through the field parsers for consistency, I think we may still need an explicit null/undefined check here so that we don't artificially create `false`-valued metadata fields that aren't actually present on the item's metadata.